src/writers/Oaipmh.php

```diff
@@ -133,6 +133,8 @@ public function writeMetadataFile($metadata, $path, $overwrite = true)
     public function normalizeFilename($string)
     {
         $string = urldecode($string);
+        $pattern = '#'.DIRECTORY_SEPARATOR.'#';
```

Comment:
We run MIK on Windows, and since DIRECTORY_SEPARATOR on Windows is `\`, we've been burned using DIRECTORY_SEPARATOR unless the pattern is run through PHP's `preg_quote()`, like this:

`$pattern = preg_quote('#' . DIRECTORY_SEPARATOR .  '#');`

I can test with that code if you want, report back, and if it works on Windows and Linux, I'll merge.